### PR TITLE
[APP-6551] - support passing scroll to bottom button

### PR DESCRIPTION
**Pull request by awlui**
## Description
Support passing renderToScrollBottomButton

### CODE REVIEW

File: src/modules/Channel/components/MessageList/index.tsx [hunk 1]
```diff
@@ -173,7 +176,22 @@ export const MessageList = ({
...
+        payload: { channel: currentGroupChannel },
+      });
+    }
+    setInitialTimeStamp(null);
+    setAnimatedMessageId(null);
+    setHighLightedMessageId(null);
+  };
 
+  const isUnreadNotificationButtonDisplayed = !!((!isScrollBottom || hasMoreNext) && (unreadSince || unreadSinceDate));
+  const isScrollToBottomButtonDisplayed = scrollBottom > SCROLL_BOTTOM_PADDING;
```
Comment (mtdurling) [inline src/modules/Channel/components/MessageList/index.tsx:194]: nit: I feel like this is easier to parse:

```
const isUnreadNotificationButtonDisplayed = (!isScrollBottom || hasMoreNext) && (!!unreadSince || !!unreadSinceDate);
 ```

Comment (mtdurling) [inline src/modules/Channel/components/MessageList/index.tsx:194]: Are these separate things? Scroll to bottom and unread notification button?

File: src/modules/Channel/components/MessageList/index.tsx [hunk 2]
```diff
@@ -173,7 +176,22 @@ export const MessageList = ({
   if (allMessagesFiltered.length < 1) {
     return renderPlaceholderEmpty();
   }
+  const handleUnreadCountClick = () => {
+    if (scrollRef?.current) scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
```
Comment (mtdurling) [inline src/modules/Channel/components/MessageList/index.tsx:180]: This functionality doesn't exist else where? Feels like we're re-inventing the wheel here.

File: src/modules/Channel/components/MessageList/index.tsx (hunk 1 above)
Comment (awlui) [reply src/modules/Channel/components/MessageList/index.tsx:194]: they are essentially the same thing in our mocks but not in uikit

File: src/modules/Channel/components/MessageList/index.tsx (hunk 2 above)
Comment (awlui) [reply src/modules/Channel/components/MessageList/index.tsx:180]: This is uikit code that I'm extracting to be re-used in the render prop. I don't think we should refactor their code beyond whats necessary

**Review by mtdurling — APPROVED** [review]

**Pull request merged**